### DISCUSSION TITLE
How to include other future forecasts in model?

### DISCUSSION THREAD

**Discussion by quannabe** [winedarksea/AutoTS#118] [Q&A]
For example, I have a time series that I am trying to predict which represents electricity demand. I also have other series that contain weather forecast data (temp, % cloud cover, etc.). The weather forecast extends 7 days from today.

So, I am wondering how I can build a model and generate electricity demand forecasts for the next 7 days which will also take into account the weather forecasts which cover the same time frame. 

I have been able to include the weather forecast, but I am getting electricity demand forecasted for days 7-14, which is a week beyond what I would like. 

I'm sure I am missing some config step or calling the model incorrectly. Any thoughts?

**Comment by winedarksea** [comment]
Look at [this section of the tutorial](https://github.com/winedarksea/AutoTS/blob/master/extended_tutorial.md#adding-regressors-and-other-information) which covers regressors.
Make sure that the date indexes of the regressors are aligned properly and that the length is equal to your input `forecast_length`.

Hard to say for sure without seeing your code.

**Reply by quannabe** [reply]
Cool, I think this is what I was looking for but will need to dig in and try this myself. 

Much appreciated and I def. didn't expect a quick response on a lazy Sunday morning :)

**Reply by winedarksea** [reply]
Well rain ruined my other plans so you were in luck! Let me know how it goes...
